lib/IRGen/IRGenDebugInfo.cpp

```diff
@@ -400,7 +400,9 @@ class IRGenDebugInfoImpl : public IRGenDebugInfo {
         }
 
         SmallVector<char, 64> Buf;
-        StringRef Name = (VD->getName().str() + Twine(Kind)).toStringRef(Buf);
+        // TODO: Handle special names
```

Comment:
This is handled in #9979 by calling through to `userFacingStr` once it exists. That's why I put the TODO there